staging/src/k8s.io/pod-security-admission/manifests/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "webhook.podsecurity-webhook.svc.cluster.local"
+webhooks:
+  - name: "webhook.podsecurity-webhook.svc.cluster.local"
+    rules:
+      - apiGroups: [""]
+        apiVersions: ["v1"]
+        operations: ["CREATE", "UPDATE"]
+        resources:
+          - cronjobs
+          - daemonsets
+          - deployments
+          - jobs
+          - namespaces
+          - pods
+          - pods/ephemeralcontainers
+          - podtemplates
+          - replicasets
+          - replicationcontrollers
+          - statefulsets
+        scope: "Namespaced"
```

Comment:
add a namespaceSelector excluding the namespace containing this webhook service/deployment/pods so we don't create a cycle that prevents bringing the webhook back up

```yaml
namespaceSelector:
  matchExpressions:
  - key: kubernetes.io/metadata.name
    operator: NotIn
    values: ["podsecurity-webhook"]
```